string timestamp
---
bool success
string message